Makefile

```diff
@@ -186,8 +186,8 @@ godeadcode:
 	deadcode -ignore version.go:Version
 
 govarcheck:
-	echo "-- checking with varcheck"
-	varcheck .
+	echo "-- Varcheck is deprecated..."
```

Comment:
Has it just moved, or is it really deprecated?

Reply:
lol, moved. Sry. Fixing.